<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EWATextSearchUi</class>
 <widget class="QWidget" name="EWATextSearchUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>38</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find text:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QToolButton" name="findNextButton">
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="ewa.qrc">
       <normaloff>:/images/skip_forward.png</normaloff>:/images/skip_forward.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="findPrevButton">
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="ewa.qrc">
       <normaloff>:/images/skip_backward.png</normaloff>:/images/skip_backward.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="csCheckBox">
     <property name="text">
      <string>Case sensetive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="wwCheckBox">
     <property name="text">
      <string>Whole wordds</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ewa.qrc"/>
 </resources>
 <connections/>
</ui>
